client/src/test/java/org/spine3/convert/StringifiersShould.java

```diff
@@ -367,7 +369,7 @@ public void convert_integer_to_string() {
         final Integer integerToConvert = 1;
         final String convertedValue =
                 new Stringifiers.IntegerStringifier().convert(integerToConvert);
```

Comment:
@IlliaShepilov,  can we use `Ints.stringConverter()` instead? There's also `Longs.stringConverter()`.